core/tests/upgrade-test/tests/tester.ts

```diff
@@ -37,7 +37,7 @@ export class Tester {
             ethWallet = new ethers.Wallet(process.env.MASTER_WALLET_PK!);
         }
         ethWallet = ethWallet.connect(ethProvider);
-        const web3Provider = new zksync.Provider(process.env.ZKSYNC_WEB3_API_URL || process.env.API_WEB3_JSON_RPC_HTTP_URL ||  "http://localhost:3050");
+        const web3Provider = new zksync.Provider(process.env.ZKSYNC_WEB3_API_URL || process.env.API_WEB3_JSON_RPC_HTTP_URL || "http://localhost:3050");
```

Comment:
please use data from the config.

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2515/commits/315b7c813752d5733d8ee85d0706f133a7d8c5da